cernopendata/modules/fixtures/data/records/cms-derived-nanoaod-outreach-higgstautau.json

```diff
@@ -0,0 +1,2693 @@
+                "variable": "MET_phi"
+            },
+            {
+                "description": "Number of stored generator particles",
+                "type": "unsigned int",
+                "variable": "nGenPart"
+            },
+            {
+                "description": "Transverse momentum of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_pt"
+            },
+            {
+                "description": "Pseudorapidity of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_eta"
+            },
+            {
+                "description": "Azimuth of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_phi"
+            },
+            {
+                "description": "Mass of the generator particles",
+                "type": "float[nGenPart]",
+                "variable": "GenPart_mass"
+            },
+            {
+                "description": "Particle identification number of the generator particles",
+                "type": "int[nGenPart]",
+                "variable": "GenPart_pdgId"
+            }
+        ],
+        "date_published": "2019",
+        "distribution": {
+            "formats": [
+                "root"
+            ],
+            "number_events": 476963,
+            "number_files": 1,
+            "size": 196517149
+        },
+        "experiment": "CMS",
+        "files": [
+            {
+                "checksum": "adler32:2b025925",
+                "size": 196517149,
+                "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/HiggsTauTauNanoAODOutreachAnalysis/GluGluToHToTauTau.root"
+            }
+        ],
+        "license": {
+            "attribution": "CC0"
+        },
+        "methodology": {
+            "description": "The datasets were produced with the software available in the record linked below.",
+            "links": [
+                {
+                    "recid": "12340"
+                }
+            ]
+        },
+        "publisher": "CERN Open Data Portal",
+        "recid": "12351",
+        "title": "GluGluToHToTauTau dataset in reduced NanoAOD format for education and outreach",
+        "type": {
+            "primary": "Dataset",
+            "secondary": [
+                "Derived"
+            ]
+        },
+        "use_with": {
+            "description": "This dataset can be used with the following analysis:",
+            "links": [
+                {
+                    "recid": "12350"
+                }
+            ]
+        },
+        "validation": {
+            "description": "<p>These data were derived from the primary datasets linked below using only the validated runs. No further validation was done for the output.</p>",
+            "links": [
+                {
+                    "recid": "8001"
+                }
+            ]
+        }
+    },
```

Comment:
It would be good to have:

```
"relations": [
      {
        "description": "This dataset was derived from:", 
        "recid": "8001", 
        "title": "/GluGluToHToTauTau_M-125_8TeV-powheg-pythia6-tauPolarOff/Summer12_DR53X-PU_S10_START53_V19-v1/AODSIM", 
        "type": "isChildOf"
      }
```
to indicate the direct relation. Now it is visible through "validation", but it would be better to have explicitly it (also for any eventual future work on the portal content display relying on parent-child relation of dataset)

Reply:
Thanks for the review and the comments! Good point here, I've added the respective section in each record.